File: coinflip.py
import random

def coin_flip_simulation(percobaan):
    angka = 0 
    gambar = 0 

    for _ in range(percobaan):
        flip = random.choice(['angka', 'gambar'])
        if flip == 'angka':
            angka += 1
        else:
            gambar += 1

    print(f"Angka : {angka}/{percobaan}")
    print(f"Gambar : {gambar}/{percobaan}")
    print(f"Probabilitas Angka: {angka / percobaan:.6f}")
    print(f"Probabilitas Gambar: {gambar / percobaan:.6f}")

coin_flip_simulation(1000000)

File: coinflip.js
function coinFlipSimulation(percobaan) {
  let angka = 0;
  let gambar = 0;

  for (let i = 0; i < percobaan; i++) {
    let flip = Math.random() < 0.5 ? "angka" : "gambar";
    if (flip === "angka") {
      angka++;
    } else {
      gambar++;
    }
  }

  console.log(`Angka (angka): ${angka}/${percobaan}`);
  console.log(`Gambar (gambar): ${gambar}/${percobaan}`);
  console.log(`Probabilitas Angka: ${(angka / percobaan).toFixed(6)}`);
  console.log(`Probabilitas Gambar: ${(gambar / percobaan).toFixed(6)}`);
}

coinFlipSimulation(1000000);
